pkg/services/libraryelements/database.go

```diff
@@ -250,9 +250,7 @@ func getLibraryElements(c context.Context, store db.DB, cfg *setting.Cfg, signed
 		builder.Write(getFromLibraryElementDTOWithMeta(store.GetDialect()))
 		builder.Write(" INNER JOIN dashboard AS dashboard on le.folder_id = dashboard.id AND le.folder_id <> 0")
 		writeParamSelectorSQL(&builder, params...)
-		if signedInUser.OrgRole != org.RoleAdmin {
```

Comment:
Note: this is not a part of the test update, but we shouldn't be filtering by basic roles like this, all filtering should be done by the query returned by `WriteDashboardPermissionFilter`.